dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/TaskInstanceController.java

```diff
@@ -95,11 +110,20 @@ public Result queryTaskListPaging(@ApiIgnore @RequestAttribute(value = Constants
                                       @RequestParam("pageNo") Integer pageNo,
                                       @RequestParam("pageSize") Integer pageSize) {
 
-        logger.info("query task instance list, project name:{},process instance:{}, search value:{},task name:{}, executor name: {},state type:{}, host:{}, start:{}, end:{}",
-                projectName, processInstanceId, searchVal, taskName, executorName, stateType, host, startTime, endTime);
+        logger.info("query task instance list, projectName:{}, processInstanceId:{}, processInstanceName:{}, search value:{}, taskName:{}, executorName: {}, stateType:{}, host:{}, start:{}, end:{}",
+                StringUtils.replaceNRTtoUnderline(projectName),
+                processInstanceId,
+                StringUtils.replaceNRTtoUnderline(processInstanceName),
+                StringUtils.replaceNRTtoUnderline(searchVal),
+                StringUtils.replaceNRTtoUnderline(taskName),
```

Comment:
if taskName is null ,will throw
 java.lang.NullPointerException: null

Reply:
I add a null operation in StringUtils.replaceNRTtoUnderline